bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/aemform/AEMForm.java

```diff
@@ -335,4 +339,16 @@ default String getExportedType() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns a list of HTML items that should be included in the form
+     *
+     * @return list of page items
+     * @since com.adobe.cq.forms.core.components.models.aemform 0.0.1
+     */
+    @Nullable
+    @JsonIgnore
+    default List<HtmlPageItem> getHtmlPageItems() {
+        throw new UnsupportedOperationException();
```

Comment:
empty list should be returned here for backward compatability.

Reply:
+1